packages/webdriverio/src/commands/browser/setTimeout.ts

```diff
@@ -43,19 +43,19 @@ export default async function setTimeout(timeouts) {
 
     const implicit = timeouts.implicit
     // Previously also known as `page load` with JsonWireProtocol
-    const pageLoad = timeouts['page load'] || timeouts.pageLoad
+    const pageLoad = (timeouts as any)['page load'] || timeouts.pageLoad
     const script = timeouts.script
 
     /**
      * JsonWireProtocol action
      */
     if (!this.isW3C) {
         return Promise.all([
-            isFinite(implicit) && this.setTimeouts('implicit', implicit),
-            isFinite(pageLoad) && this.setTimeouts('page load', pageLoad),
-            isFinite(script) && this.setTimeouts('script', script),
+            isFinite(implicit) && (this.setTimeouts as any)('implicit', implicit),
```

Comment:
consider refactoring `(this.setTimeouts as any)` into a const to reduce duplication or even better if it's possible, change the type of `this.setTimeouts`